CHANGELOG.md

```diff
@@ -1315,8 +1330,10 @@ Install with `npm install onfido-sdk-ui@0.12.0-rc.1`
 
 - NPM (commonjs2) style of importing the library now works
 
-[next-version]: https://github.com/onfido/onfido-sdk-ui/compare/8.1.1...development
-[8.1.1]: https://github.com/onfido/onfido-sdk-ui/compare/8.1.0...8.1.1
+[next-version]: https://github.com/onfido/onfido-sdk-ui/compare/null...development
+[null]: https://github.com/onfido/onfido-sdk-ui/compare/9.0.0-rc.1...null
```

Comment:
weird null link, but I guess this goes away when we do the full release

Reply:
Good catch, fixed it.